### PR TITLE
check if data and env format

### PR DESCRIPTION
**Pull request by cheyang**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1337)

### CODE REVIEW

File: components/arena/docker/job_generator.py [hunk 1]
```diff
@@ -52,11 +52,18 @@ def generate_options(args):
...
+        else:
+            logging.info("--data={0} is illegal, skip.".format(d))
 
     if len(env) > 0:
       for e in env:
-        options.append("--env={0}".format(e))
+        if "=" in e:
+            options.append("--env={0}".format(e))
+        else:
+            logging.info("--env={0} is illegal, skip.".format(d))
```
Comment (animeshsingh) [inline components/arena/docker/job_generator.py]: this should be 
`logging.info("--env={0} is illegal, skip.".format(e) `
?

Comment (cheyang) [reply components/arena/docker/job_generator.py]: Thank you for spotting it.

Comment (cheyang) [reply components/arena/docker/job_generator.py]: Fix it. Done.

**Pull request merged**